src/CONST.ts

```diff
@@ -16,6 +16,8 @@ import type PlaidBankAccount from './types/onyx/PlaidBankAccount';
 const EMPTY_ARRAY = Object.freeze([]);
 const EMPTY_OBJECT = Object.freeze({});
 
+const DEFAULT_NUMBER_ID = 0;
```

Comment:
Curious why we add a const and then use it only once in the object instead of just defining this directly in the object. IMO the less indirection, the better